<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pcc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="codeTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>841</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="resultTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>841</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tokenTab">
     <attribute name="title">
      <string>Tokens</string>
     </attribute>
     <widget class="QListWidget" name="tokenListWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>841</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lexerTab">
     <attribute name="title">
      <string>Lexical Errors</string>
     </attribute>
     <widget class="QListWidget" name="lexerListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>841</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="parserTab">
     <attribute name="title">
      <string>Parsing Errors</string>
     </attribute>
     <widget class="QListWidget" name="parserListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>841</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>24</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>MainClass</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionCompile"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuBuild"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
